### PR TITLE
Updated the builders for glove vectors

### PR DESCRIPTION
**Pull request by Joshua-Chin**

### CODE REVIEW

File: conceptnet5/builders/raw_glove.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (rspeer) [inline conceptnet5/builders/raw_glove.sh]: I wouldn't say that a shell script belongs here, especially not one that has to be run out of the current directory. Maybe this belongs as a Ninja build step?

**Comment by Joshua-Chin** [comment]
I can move the raw_glove.sh code into the ninja build as soon as the two branches are merged.

**Comment by rspeer** [comment]
Which two branches? ninja-build is already merged, isn't it?

**Comment by rspeer** [comment]
Ah, as in I should merge version5.4 into retrofitting? Sure, it's about time for that.

**Comment by whitten** [comment]
So for those of us getting up to speed, I assume a "glove" is a data structure.
What is it used for? Is there any documentation re its contract with other objects?

**Comment by rspeer** [comment]
Hello whitten. What you're looking at is an experimental branch of the code.

GloVe (http://nlp.stanford.edu/projects/glove/) is a recent project that produces word vectors that can be used for measuring semantic similarity. Recently, a technique called "retrofitting" has been published that allows combining an arbitrary semantic network (such as ConceptNet) with a trained collection of word vectors (such as GloVe). On this `retrofitting` branch, we're experimenting with doing this, which does seem to bring noticeable benefits.

**Pull request merged**